http/sys_hostinfo_test.go

```diff
@@ -52,7 +52,16 @@ func TestSysHostInfo(t *testing.T) {
 		t.Fatal("expected memory info")
 	}
 
-	// Query against the standby, should error
+	// Check we're standby
+	healthResp, err := cores[1].Client.Sys().Health()
```

Comment:
Bear in mind that it may not be healthy at this point in time.  You might want a for loop to poll until it doesn't return an error or a timeout is reached.

Reply:
TestWaitActive will wait for the active node to get its lock.  There's still a delay between that and the standby noticing, reading the active node from storage, and being able to forward requests to it.  Moreover the standby is a distinct process and in a constrained environment like circleci it could be lagging far behind, still just coming up.

Reply:
s/process/goroutine/